mkdocs.yml

```diff
@@ -22,17 +22,18 @@ nav:
             - Mac OSX 10.11+: installation/osx.md
         - Tango: installation/tango.md
         # - (Legacy) OneClick Install: installation/one-click.md
-
-    - Autolab API:
-        - Overview: api-overview.md
-        - Reference: api-interface.md
-        - Command Line Interface: command-line-interface.md
-        - Managing Authorized Apps: api-managing-authorized-apps.md
+    - Reference: 
+        - Reference Documentation: reference.md
+        - Autolab Frontend API:
+            - Overview: api-overview.md
+            - Reference: api-interface.md
```

Comment:
Thought: For Reference/Autolab Frontend API/Reference, would it be better if we could name it Endpoints or something other than Reference? The nested "Reference" feels a bit redundant to me. But I'm up to the majority's opinion.

Reply:
I think endpoints is a suitable change